<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hardConfigure</class>
 <widget class="QDialog" name="hardConfigure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1305</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hardConfigure</string>
  </property>
  <widget class="QGroupBox" name="Rx_Gain">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>290</y>
     <width>239</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Gain</string>
   </property>
   <widget class="QSpinBox" name="spinBox_1">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>57</width>
      <height>29</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>1~100</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>dB</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Tx_Gain">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>290</y>
     <width>249</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Gain</string>
   </property>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>57</width>
      <height>29</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>1~100</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>dB</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="CCA_Threshold">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>154</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>CCA Threshold</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>129</width>
      <height>29</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Must be a positive integer</string>
    </property>
    <property name="text">
     <string>65535</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>609</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="Rx_Sample_Rate">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>200</y>
     <width>239</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Sample Rate</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_11">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>129</width>
      <height>29</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Positive number</string>
    </property>
    <property name="text">
     <string>12.5</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>25</y>
      <width>66</width>
      <height>29</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>SPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MSPS</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>199</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>State</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>165</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>Send</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_9">
       <property name="text">
        <string>Receive</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>721</x>
     <y>61</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>USRP Source</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="Ack_Timeout">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>181</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>ACK Timeout</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>129</width>
      <height>29</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Must be a positive integer</string>
    </property>
    <property name="text">
     <string>1250</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>21</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>us</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>609</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="Scale_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>131</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>DIFS Time</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_12">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Must be a positive integer</string>
    </property>
    <property name="text">
     <string>34</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>21</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>us</string>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>985</x>
     <y>660</y>
     <width>251</width>
     <height>29</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Mac Layer</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="Dot11_Fragmentation_Threshold">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>PSDU Length</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>129</width>
      <height>29</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>1~4055</string>
    </property>
    <property name="text">
     <string>4095</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>byte(s)</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>1007</x>
     <y>61</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>USRP Sink</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>UHD Set</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>11</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Choose File</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>/Photo/grt.jpg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Scale">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>125</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>SIFS Time</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Must be a positive integer</string>
    </property>
    <property name="text">
     <string>16</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>21</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>us</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Rx_Center_Frequency">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>110</y>
     <width>239</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Center Frequency</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>129</width>
      <height>29</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Positive number</string>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>2.457</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>25</y>
      <width>60</width>
      <height>29</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>HZ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GHZ</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Rate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>221</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Rate Choose</string>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>191</width>
      <height>236</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_1">
       <property name="text">
        <string>6Mbps (BPSK 1/2)</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>9Mbps (BPSK 3/4)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>12Mbps (QPSK 1/2)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>18Mbps (QPSK 3/4)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_5">
       <property name="text">
        <string>24Mbps (16 QAM 1/2)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_6">
       <property name="text">
        <string>36Mbps (16 QAM 3/4)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_7">
       <property name="text">
        <string>48Mbps (64 QAM 2/3)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_8">
       <property name="text">
        <string>54Mbps (64 QAM 3/4)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Phy Layer</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>130</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Source Mac Address:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="inputMask">
       <string>HH:HH:HH:HH:HH:HH</string>
      </property>
      <property name="text">
       <string>ce:ca:11:ce:ca:11</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Dest Mac Address:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="inputMask">
       <string>HH:HH:HH:HH:HH:HH</string>
      </property>
      <property name="text">
       <string>ce:ca:22:ce:ca:22</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="Tx_Center_Frequency">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>110</y>
     <width>249</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Center Frequency</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>129</width>
      <height>29</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Positive number</string>
    </property>
    <property name="text">
     <string>2.457</string>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string extracomment="txcenterfrequency"/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_1">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>25</y>
      <width>60</width>
      <height>29</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>HZ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GHZ</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>40</y>
     <width>20</width>
     <height>611</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>50</y>
     <width>16</width>
     <height>311</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="Tx_Sample_Rate">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>200</y>
     <width>249</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Sample Rate</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>25</y>
      <width>129</width>
      <height>29</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Positive number</string>
    </property>
    <property name="text">
     <string>12.5</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>25</y>
      <width>66</width>
      <height>29</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>SPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MSPS</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>430</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>make_device</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>430</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>430</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>hardConfigure</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1121</x>
     <y>678</y>
    </hint>
    <hint type="destinationlabel">
     <x>784</x>
     <y>621</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
